docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -20,85 +20,124 @@ zkApps can now be deployed to Berkeley Testnet.
+), linting [ES Lint](https://eslint.org/), and more.
 
-### Install Mina zkApp CLI
+## Install Mina zkApp CLI
 
 ```sh
 npm install -g zkapp-cli
 ```
 
-#### Dependencies:
+Dependencies:
 
 - NodeJS 16+ (or 14 using `node --experimental-wasm-threads`)
 - NPM 6+
 - Git 2+
 
-:::tip
+If you have a later version installed, install the required version using the package manager for your system: 
 
-If you have an older version installed, install a newer version using the package manager for your system: <a href="https://brew.sh/">Homebrew</a> (Mac), <a href="https://chocolatey.org/">Chocolatey</a> (Windows), or apt/yum/etc (Linux). On Linux, you may need to install a recent NodeJS version via NodeSource (<a href="https://github.com/nodesource/distributions#debinstall">deb</a> or <a href="https://github.com/nodesource/distributions#rpminstall">rpm</a>), as recommended by the NodeJS Project.
+- MacOs [Homebrew](https://brew.sh/)
 
-:::
+- Windows [Chocolatey](https://chocolatey.org/)
+
+- Linux (apt, yum, and others)
+
+    As recommended by the Node.js project, you might need to install a recent Node.js version using NodeSource binary distributions: [Debian](https://github.com/nodesource/distributions#debinstall), [rpm](https://github.com/nodesource/distributions#rpminstall).
+
+## Start a project
+
+Now that you have the Mina zkApp CLI installed, you can start with an example or start your own project.
+
+### Option A: Start with an example (recommended)
+
+Examples are based on the standard project structure and provide additional files in the `/src` directory.
+
+1. Create the sudoku example project: `zk example sudoku`
+
+  The created project includes the example files (the smart contract) in the project's `src/` directory. 
+  
+  To see the files that were created, change to the `sudoku` directory and run the `ls` command or open the directory in a code editor, such as VS Code.
+
+1. Run tests and see the tests pass: `npm run test`  
+  
+  To rerun tests automatically when you save changes to your code, run the tests in watch mode with `npm run testw`.
+
+1. Build the example: `npm run build`
+
+  Compile your TypeScript into JavaScript in the project `/build` directory.
+
+1. Configure your zkApp: `zk config`
+
+  The command prompts guide you in adding a network alias to your project `config.json` file. 
+  
+  For Berkeley Testnet, use:
+  
+    - name: `berkeley`
+    - Mina GraphQL API URL: `https://proxy.berkeley.minaexplorer.com/graphql`
+    - transaction fee: `0.1`
+
+1. Fund your fee payer account.
+
+  Follow the prompts to request tMina. 
+
+1. Deploy to Testnet: `zk deploy`
+
+  Follow the prompts. For details, see [how to deploy a zkApp](how-to-deploy-a-zkapp).
+
+### Option B: Start your own project
+
+1. Create your own project: `zk project <myproj>`
+
```

Comment:
If a user runs `zk project <myproj>`, a user will get a prompt to create an accompanying UI project.
```
? Create an accompanying UI project too? …
❯ next
  svelte
  nuxt
  empty
  none
```

It would be helpful mention this step somehow. You could guide the user to either select the `none` option or use the command `zk project <myproj> --ui none`.  

This brings up the question if we should highlight the interactive(`zk project <myproj>`) or non-interactive(`zk project <myproj> --ui none`) commands as defaults in the docs. WDYT @barriebyron ?

Reply:
yes, interactive is good for learning, I like this suggestion